### ISSUE TITLE
Image description/alt-text for people who can not see

### ISSUE THREAD

**Issue by hoijui** [OPEN-NEXT/OKH-LOSH#105]
This was requested [in an issue](https://app.standardsrepo.com/MakerNetAlliance/OpenKnowHow/issues/372) for OKH v1.

Our RDF already supports it, as our image property derives from `schema:image`, which allows to reference either a URL or an [ImageObject](https://schema.org/ImageObject), which supports a lot of attributes that could serve for this.

Our TOML though, does not currently support this, so I propose to add this (or say.. to consider it).

**Comment by moedn** [comment]
hmm the link to the original issue is dead

**Comment by moedn** [comment]
+1 :)

**Comment by hoijui** [comment]
> hmm the link to the original issue is dead

yep, the whole site they used to host their standard (including all the discussion and issues) is dead:

> The domain name
StandardsRepo.com
is for sale!

Take this as an argument for not using git as a base!
... they switched to yet something else that is .. some custom made startup ... bla thing, btw. ;-)
